Frameworks/CoreGraphics/CGColorSpace.mm

```diff
@@ -175,15 +175,16 @@ size_t CGColorSpaceGetNumberOfComponents(CGColorSpaceRef pSpace) {
  @Status Interoperable
 */
 void CGColorSpaceRelease(CGColorSpaceRef colorSpace) {
-    CFRelease((id)colorSpace);
+    RETURN_IF(!colorSpace);
+    CFRelease(static_cast<CFTypeRef>(colorSpace));
```

Comment:
it's part of the whole CGColorSpace rewrite :)

---
In reply to: [106480069](https://github.com/Microsoft/WinObjC/pull/2249#discussion_r106480069) [](ancestors = 106480069)